notebooks/README.md

```diff
@@ -0,0 +1,38 @@
+# CNN-Annotator Repository Structure
+
+![Protocol Pipeline](../assets/repository_structure.png)
+
+### Annotating CNN training images using *Napari*:
```

Comment:
This is great too. I wonder if we can export the paper itself as markdown when we're finished with it?